src/app/zeko/circuits/indexed_merkle_tree.mli

```diff
@@ -48,7 +48,7 @@ end) : sig
   end
 
   val add_key_var :
-       ?check:Boolean.var
```

Comment:
@L-as was this check intentionally optional or can I make it required?

Reply:
it was intentional but if it's never called with None it's OK